CMakeLists.txt

```diff
@@ -865,6 +865,13 @@ cmake_dependent_option(
   "USE_CUDA OR USE_ROCM;NOT MSVC"
   OFF)
 
+cmake_dependent_option(
+  USE_CK_FLASH_ATTENTION
```

Comment:
We have environment variable - USE_CK_FLASH_ATTENTION, now we are adding cmake variable. is it requried??